Update/kake_037.txt

```diff
@@ -472,11 +472,11 @@ void main()
 			NULL, "\"Anyway, it's a shame.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 11, "ps3/s20/11/vtk1_oishi1114", 256, TRUE);
 	OutputLine(NULL, "ぜひとも今夜は、その鹿嶋桜の酒蔵を南井さんに教えてもらいたかったのに。",
-			NULL, " It would've been nice if Minai-san could've been here tonight to tell us all about that Kashima Sakura brewery. ", GetGlobalFlag(GLinemodeSp));
+			NULL, " It would've been nice if Minai-san could've been here tonight to tell us all about that Kashimazakura brewery. ", GetGlobalFlag(GLinemodeSp));
```

Comment:
```suggestion
			NULL, " It would've been nice if Minai-san could've been here tonight to tell us all about that Kashima Sakura brewery. ", GetGlobalFlag(GLinemodeSp));
```